Chapter 14/data-loaders/friend.js

```diff
@@ -6,7 +6,7 @@ const DataLoader = require('dataloader')
 module.exports = function buildDataLoader (app) {
   const dl = new DataLoader(async function fetcher (ids) {
     const secureIds = ids.map(id => SQL`${id}`)
-    const sql = SQL`SELECT * FROM Friend WHERE personId IN (${SQL.glue(secureIds, ',')})`
+    const sql = SQL`SELECT * FROM friend WHERE personId IN (${SQL.glue(secureIds, ',')})`
```

Comment:
I read your feedback here, but I fixed all the occurrences in `Pascal` case to be homogeneous - I must say that I don't have strong opinions on this, just habit, and I know it is a bikeshedding topic

Reply:
I am used to table names in lowercase, the same case was used in the queries in the book before Dataloaders introduction. That's why I made this change.